src/Tools/dotnet-monitor/CollectionRules/Actions/CollectTraceAction.cs

```diff
@@ -78,8 +81,9 @@ public CollectTraceAction(IServiceProvider serviceProvider, IEndpointInfo endpoi
                     EventPipeProvider[] optionsProviders = Options.Providers.ToArray();
                     bool requestRundown = Options.RequestRundown.GetValueOrDefault(CollectTraceOptionsDefaults.RequestRundown);
                     int bufferSizeMegabytes = Options.BufferSizeMegabytes.GetValueOrDefault(CollectTraceOptionsDefaults.BufferSizeMegabytes);
-
```

Comment:
Note that if we end up going the client way for this work, we'll need to add this configuration into the /trace post endpoint as well. That's probably desirable either way.